### PR TITLE
style: refine color vo.1

### PR DESCRIPTION
**Pull request by lavalse**
# Overview
This PR's target is a quick fixing bugs in Light theme colors.
To make our color system works better, I will still keep refining colors in other PR.

## What I've done
- Fix the color bug I found
- Update Logo in setting page and dashboard
- add Brand gradient in dashboard
- Update some code which get color from theme

## What I haven't done
Totally refine colors

## How I tested
Update , and at same time test

## Screenshot
Please use deploy website to check colors
image
image
image
image

### CODE REVIEW

**Review by rot1024 — DISMISSED** [review]

File: src/components/molecules/Dashboard/Logo.tsx [hunk 1]
```diff
@@ -1,26 +1,23 @@
...
 export interface Props {
   className?: string;
 }
 
 const Logo: React.FC<Props> = ({ className }) => {
   return (
-    <Wrapper className={className}>
-      <Icon icon="logo" />
+    <Wrapper align="center" justify="center" className={className}>
+      <StyledImg src={logoColorful} alt="Re:earth Logo" />
```
Comment (KaWaite) [inline src/components/molecules/Dashboard/Logo.tsx:13]:
```suggestion
      <StyledImg src={logoColorful} alt="Re:Earth Logo" />
```

File: src/components/molecules/Settings/Navigation/index.tsx [hunk 2]
```diff
@@ -29,7 +29,7 @@ const Navigation: React.FC<Props> = ({ team, project }) => {
   return (
     <Wrapper>
       <LogoWrapper>
-        <Icon icon="logo" size="100px" />
+        <StyledImg src={logoColorful} alt="Re:earth Logo" />
```
Comment (KaWaite) [inline src/components/molecules/Settings/Navigation/index.tsx:32]:
```suggestion
        <StyledImg src={logoColorful} alt="Re:Earth Logo" />
```

File: src/theme/colors.ts [hunk 3]
```diff
@@ -39,12 +39,9 @@ const colors = {
       success: "#00D1A2" /* Unused */,
       attention: "#D6C71C",
       error: "#FF3C53",
-      select: "#0EB28E" /* Untested/Unused */,
+      select: "#3B3CD0" /* Untested/Unused */,
```
Comment (KaWaite) [inline src/theme/colors.ts:42]: Sorry, for any colors here that you KNOW you have used can you remove the comments saying it is unused or untested in this file?
I put them there when I updated these colors to match Figma and I noticed at that point not all the colors were being used yet.

**Review by KaWaite — APPROVED** [review]
If you can make the changes I suggested and remove any unnecessary comments in the colors.ts file this looks good!

File: src/theme/colors.ts (hunk 3 above)
Comment (lavalse) [reply src/theme/colors.ts:42]: @KaWaite not all the colors are used for sure, still one and two color maybe not used. But now the colors here are the same with the design, so which is good I think. I also change some color based on our service. I will remove the commet s here

**Pull request merged**